### DISCUSSION TITLE
Customize the font used for LaTeX nodes

### DISCUSSION THREAD

**Discussion by bruno-brant** [freeplane/freeplane#481] [Help] [answered]
How can I change the font used for LaTeX? Specifically, how can I change the font to be the same as the other nodes?

The font used in LaTeX nodes is (purposefully, I believe) different from that used in other nodes. However, I find it very distracting; also, it's on a different scale/size than the other font, so at certain zoom levels I can read normal nodes but not LaTeX ones.

**Comment by fnatter** [comment, marked as answer]
hello bruno-brant,

you can easily change from with-serifs to without-serifs using \textsf{...}. Unfortunately, you cannot change
the font type and the scaling is different. The scaling could possibly be fixed (-> feature request).
Cheers and Best Regards,
Felix
